<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>midiJGwClass</class>
 <widget class="QWidget" name="midiJGwClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MidiJ Gateway</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/MidiJGw.qrc">
    <normaloff>:/midijgw/midijgw_icon.png</normaloff>:/midijgw/midijgw_icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="windowLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="midiJGwLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>MidiJ Gateway</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>ERROR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="controlsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>96</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="controlsLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="leftSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>39</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="midiControlsLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QCommandLinkButton" name="midiController1">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>54</height>
             </size>
            </property>
            <property name="text">
             <string>MidiJ - UDP</string>
            </property>
            <property name="description">
             <string>MIDI Controller</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="midiController2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>54</height>
             </size>
            </property>
            <property name="text">
             <string>MidiJ - UDP</string>
            </property>
            <property name="description">
             <string>MIDI Controller</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="midiLinesLayout">
          <item>
           <widget class="LightLine" name="midiController1Line">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::HLine</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="midLineWidth">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LightLine" name="midiController2Line">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::HLine</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="midLineWidth">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="gatewayController">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>54</height>
           </size>
          </property>
          <property name="text">
           <string>Dual Direct MIDI</string>
          </property>
          <property name="description">
           <string>Gateway Controller</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LightLine" name="gatewayControllerLine">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="midLineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="midiOutput">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>54</height>
           </size>
          </property>
          <property name="text">
           <string>LoopBE 1</string>
          </property>
          <property name="description">
           <string>MIDI Output</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="rightSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>39</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <widget class="QLabel" name="MIDI_DEBUG">
       <property name="text">
        <string>DEBUG:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="midiController1LineOLD">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDirectMidi">
   <property name="text">
    <string>DirectMidi</string>
   </property>
  </action>
  <action name="actionDualDirectMidi">
   <property name="text">
    <string>DualDirectMidi</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LightLine</class>
   <extends>QFrame</extends>
   <header>lightline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/MidiJGw.qrc"/>
 </resources>
 <connections/>
</ui>
